apps/vr-tests-react-components/src/stories/Table.stories.tsx

```diff
@@ -27,7 +27,6 @@ import {
 import { Button } from '@fluentui/react-button';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28493&commitId=516007752436576c98ba061fdca01aedf66d2e2b&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 2090 screenshots unchanged, 0 screenshots with different dimensions and 2 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex.keyboard column resizing style.chromium.png|7820| Changed|
|Table layout table.keyboard column resizing style.chromium.png|8064| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>